packages/utilities/fast-benchmarks/benchmarks/fast-element/observable/index.ts

```diff
@@ -0,0 +1,39 @@
+import { Observable, observable } from "@microsoft/fast-element";
+import { _random, adjectives, nouns } from "../../../utils/index.js";
+
+export class Pupil {
+    @observable greetMessage: string = "";
+    @observable name: string = "";
+    @observable exit: boolean = false;
+
+    constructor(firstName: string, lastName: string) {
+        const first = firstName[0].toUpperCase() + firstName.slice(1);
+        const last = lastName[0].toUpperCase() + lastName.slice(1);
+
+        this.name = `${first} ${last}`;
+        this.greetMessage = `Welcome to FAST, ${this.name} !!`;
+    }
+
+    sayGoodbye() {
+        this.greetMessage = `Goodbye ${this.name}, see you next time!`;
+    }
+}
+
+(window as any).runFunction = () => {
```

Comment:
If you add a declaration file, you can declare the global function in it so that intellisense picks it up and you don't need to `as any`. See https://stackblitz.com/edit/typescript-l4pxcy?file=global.d.ts for an example implementation.